### ISSUE TITLE
Some buttons are missing accessible names

### ISSUE THREAD

**Issue by krschau** [microsoft/devhome#1961]
### Dev Home version

0.7

### Steps to reproduce the bug

Try to read certain buttons with Narrator. They just say "button", but should have a descriptive name.
https://task.ms/46728351
https://task.ms/46742305
https://task.ms/46727423
